app/views/layouts/application.html.erb

```diff
@@ -3,7 +3,15 @@
   <head>
     <title><%= content_for(:page_title) || "Samson" %></title>
     <%= stylesheet_link_tag    "application", media: "all" %>
+    <% Samson::Hooks.plugins.map(&:stylesheets).flatten.each do |css| %>
+      <%= stylesheet_link_tag asset_path(css) %>
+    <% end %>
+
     <%= javascript_include_tag "application" %>
+    <% Samson::Hooks.plugins.map(&:javascripts).flatten.each do |javascript| %>
```

Comment:
I can't see any `.erb.js` in this PR, shouldn't they be [here](https://github.com/zendesk/samson/tree/fneves/buddy_request_plugin/plugins/flowdock/app/assets/javascripts) or [here](https://github.com/zendesk/samson/tree/fneves/buddy_request_plugin/plugins/flowdock/app/views/samson_flowdock)?